### PR TITLE
Feature/final refactor

### PR DESCRIPTION
**Pull request by justincanthony**
The PR would like to merge the final updates and refactoring to the main branch.

**JS Updates**
- A function was added to delete messages
- Code has been refactored with all functions working properly
- Variables have been renamed and updated with appropriate content

**CSS Updates**
- Flexbox has been applied to several elements for ease of arrangement
- Style updates to buttons to match page themes
- Font color change on the alert message 

**HTML UPDATEs**
- < div > elements added to engage flexbox usability 
- Class names updated to match css styling rules

**Image Directory Added**
- I added an image directory to potentially use for adding images to the README

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -12,6 +12,15 @@ body {
   text-align: center;
 }
 
+.button {
+  background: #134d71;
```
Comment (dreardon1021) [inline styles.css:16]: Not a big deal but make this `background-color to be more semantic

File: index.html [hunk 2]
```diff
@@ -6,30 +6,32 @@
...
+        <button class ="receive-message button quicksand" type ="button">Recieve Message</button>
+        <button class ="delete-message button hidden quicksand" type ="button">Delete Message</button>
       </div>
     </form>
     <section class ="image-box">
       <img class ="gong-image" src ="assets/meditate.svg" alt ="gong" />
-      <p class ="quicksand hidden" id ="message-display"></p>
+    <br />
+    <br />
+      <div class = image-box-flex>
```
Comment (dreardon1021) [inline index.html:32]: You don't need this div. If you go to dev tools and inspect this div it isn't the same size as the white box thus making it not perfectly center relative to the white image box. Try removing the div and adding `image-box-flex` to the class list above in the section element

Remove the breaks, and then play around with justify-content, align-items to try and properly center the text in the image box

**Review by dreardon1021 — APPROVED** [review]
Looks good. A few minor changes to think about but will approve. Functionality looks good to the project spec!

Let me know if you have questsions

File: styles.css (hunk 1 above)
Comment (justincanthony) [reply styles.css:16]: OK- Thank you for pointing that out. I will make the changes.

File: index.html (hunk 2 above)
Comment (justincanthony) [reply index.html:32]: I knew this was a bad example of letting HTML do whatever I wanted.  I was getting some grief from the image and didn't have time to noodle around without finishing the rest of the project specs on time. Now that everything is in place, I will attempt to apply the correct method. Thank you for catching that!

Comment (justincanthony) [reply index.html:32]: I am going to have to spend some more time with this in order to get it to display correctly. I will merge the original _sloppy_ HTML version, and attempt to make progress with it later. Thanks again for the feedback, It has been incredibly helpful!

**Pull request merged**